packages/core/src/lib/features/FeatureManager.ts

```diff
@@ -43,11 +44,20 @@ export class FeatureManager {
     variables: new Set<string>(),
     launchUrl: new Array<string>(),
   };
+  delegationService = {
+    imports: new Set<string>(),
+    constructor: new Array<string>(),
+  };
 
   /**
    * Builds a new intance from a TwaManifest.
    */
   constructor(twaManifest: TwaManifest) {
+    if (twaManifest.features.locationDelegation &&
+        twaManifest.features.locationDelegation.enabled) {
```

Comment:
I just realized that this can be written as:

```
if (twaManifest.features.locationDelegation?.enabled)
```

Look at this - https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

I've always found that feature pretty cool (you can also change line 61).